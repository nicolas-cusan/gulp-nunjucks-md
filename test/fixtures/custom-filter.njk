<!doctype html>
<html>
<head lang="en">
    <meta charset="utf-8">
    <title>Test nunjucks project</title>
</head>
<body>
    <h1>{{ globalTitle }}</h1>
    <h2>{{ ''|defaultName('Peter Smith') }}</h2>
    <h3>{{ 'jane cortwel'|defaultName }}</h3>
</body>
</html>
