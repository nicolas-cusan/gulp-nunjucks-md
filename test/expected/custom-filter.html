<!doctype html>
<html>
<head lang="en">
    <meta charset="utf-8">
    <title>Test nunjucks project</title>
</head>
<body>
    <h1>Test nunjucks project</h1>
    <h2>PETER SMITH</h2>
    <h3>JANE CORTWEL</h3>
</body>
</html>
